src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -3150,7 +3151,7 @@ static void updateEntity
   cerP->contextElement.entityId.fill(entityId, entityType, "false");
 
   /* If the vector of Context Attributes is empty and the operation was DELETE, then delete the entity */
-  if (strcasecmp(action.c_str(), "delete") == 0 && ceP->contextAttributeVector.size() == 0)
+  if ((action == ActionTypeDelete) && (ceP->contextAttributeVector.size() == 0))
```

Comment:
So, we had a `strcasecmp` here before ...
Is this a possible problem for backward compatibility? (depends on the action-type-parse function).

Reply:
Moved to new method `parseActionTypeV1()`, where `strcasecmp()` is used.

NTC (I guess)

Reply:
Yeah, I saw.

NTC